v21.1/connection-parameters.md

```diff
@@ -88,7 +88,20 @@ Parameter | Description | Default value
 `sslrootcert` | Path to the [CA certificate](cockroach-cert.html), when `sslmode` is not `disable`. | Empty string.
 `sslcert` | Path to the [client certificate](cockroach-cert.html), when `sslmode` is not `disable`. | Empty string.
 `sslkey` | Path to the [client private key](cockroach-cert.html), when `sslmode` is not `disable`. | Empty string.
-`options` | Additional command-line options to be passed to the server. Specify the cluster name when connecting to CockroachCloud free-tier clusters by setting `options=--cluster=<cluster name>`. | Empty string
+`options` | [Additional options](#supported-options-parameters) to be passed to the server. | Empty string
+
+#### Supported `options` parameters
+
+CockroachDB supports the following `options` parameters. After the first `options` parameter is specified, any additional parameters in the same connection string must be separated by a space.
+
+Parameter | Description
+----------|-------------
+`--cluster=<cluster name>` | Specifies the cluster name when connecting to [CockroachCloud free-tier clusters](connect-to-the-database-cockroachcloud.html#connect).
+`-c <session_variable>=<value>` | <span class="version-tag">New in v21.1:</span> Sets a [session variable](set-vars.html) for the SQL session.
+
+{{site.data.alerts.callout_info}}
+Note that some drivers require space characters to be properly encoded in URL connection strings. For example, spaces in [a JDBC connection string](https://jdbc.postgresql.org/documentation/head/connect.html#connection-parameters) must specified as `%20`.
```

Comment:
Isn't it more than just spaces? E.g. `options=--cluster=<cluster name>` would require the second equals sign to be URL encoded.

Reply:
You're right - it could be more than just spaces. I've updated the description to use spaces just as an example.

IDK about equal signs though... At least in the case of the JDBC driver (https://jdbc.postgresql.org/documentation/head/connect.html#connection-parameters), the equal signs don't need to be URL-encoded. Only spaces do.